examples/bss_example.py

```diff
@@ -129,14 +143,20 @@ def convergence_callback(Y):
     ###########
     # The STFT needs front *and* back padding
 
-    # shape == (n_chan, n_frames, n_freq)
     X = np.array([pra.stft(ch, L, L, transform=np.fft.rfft, zp_front=L//2, zp_back=L//2) for ch in mics_signals])
     X = np.moveaxis(X, 0, 2)
 
-    # Run AuxIVA
-    Y = pra.bss.auxiva(X, n_iter=30, proj_back=True, callback=convergence_callback)
-
-    # run iSTFT
+    # Run BSS
+    bss_type = args.algo
```

Comment:
An error will occur if the user provides a parameter that is neither 'auxiva' nor 'ilrma'. Restricting at the input (as suggested above) will solve this.

Reply:
Fixed